modules/acknowledgements/jsx/main.js

```diff
@@ -0,0 +1,55 @@
+/* global loadAcknowledgementForm */
```

Comment:
You shouldn't be mixing up React in all this jQuery hell. Or rather you shouldn't be coding all the views in `jQuery`. Or rather you shouldn't be using jQuery at all for anything but ajax calls. 

Take a look at https://github.com/aces/Loris/tree/17.1-dev/modules/imaging_uploader/jsx to see how a modern Loris module should be constructed

Also, I suggest to watch the first couple of videos from this series https://laracasts.com/series/do-you-react
and this free course https://reacttraining.com/online/react-fundamentals